docs/advanced-configuration.rst

```diff
@@ -549,3 +549,643 @@ See ``conf/settings-defaults.cfg`` for an exhaustive list:
+   * *Stop your instance admin.*
+   * *Change in the settings*
+   * *Re-run the server.*
+
+**Translations**
+::
+
+   MODELTRANSLATION_LANGUAGES = ('en', 'fr', 'it', 'es')
+
+Languages of your project. It will be used to generate fields for translations. (ex: description_fr, description_en)
+
+   *You won't be able to change it easily, avoid to add any languages and do not remove any.*
+
+**Options admin**
+=================
+**Map config**
+::
+
+    LEAFLET_CONFIG['TILES'] = [
+        ('Scan', '//wxs.ign.fr/<key>/wmts?LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN-EXPRESS.STANDARD&EXCEPTIONS=image/jpeg&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
+         '&copy; IGN - GeoPortail'),
+        ('Ortho', '//wxs.ign.fr/<key>/wmts?LAYER=ORTHOIMAGERY.ORTHOPHOTOS&EXCEPTIONS=image/jpeg&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
+         '&copy; IGN - GeoPortail'),
+        ('Cadastre', '//wxs.ign.fr/<key>/wmts?LAYER=CADASTRALPARCELS.PARCELS&EXCEPTIONS=image/jpeg&FORMAT=image/png&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
+         '&copy; IGN - GeoPortail'),
+        ('OSM', 'http://{s}.tile.osm.org/{z}/{x}/{y}.png', '&copy; OSM contributors'),
+    ]
+
+    LEAFLET_CONFIG['OVERLAYS'] = [
+        ('Cadastre',
+         '//wxs.ign.fr/<key>/wmts?LAYER=CADASTRALPARCELS.PARCELS&EXCEPTIONS=text/xml&FORMAT=image/png&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=bdparcellaire_o&TILEMATRIXSET=PM&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}',
+         '&copy; IGN - GeoPortail'),
+    ]
+
+Configuration of the tiles.
+
+    *If you want to change it,*
+    *Change the array like that :*
+
+    ::
+
+        LEAFLET_CONFIG['TILES'] = [('NAME_OF_TILE', 'URL', 'COPYRIGHT'), ...]
+
+    *It's the same for the overlay but use only transparent tiles*
+
+|
+
+::
+
+    LEAFLET_CONFIG['MAX_ZOOM'] = 19
+
+You can define the max_zoom the user can zoom for all tiles.
+
+    *It can be interesting when your tiles can't go to a zoom. For example opentopomap is 17.*
+
+**Enable Apps**
+::
+
+    FLATPAGES_ENABLED = True
+
+Show Flatpages on menu or not. Flatpages are used in Geotrek Rando.
+
+|
+
+::
+
+    TOURISM_ENABLED = True
+
+Show TouristicContents and TouristicEvents on menu or not.
+
+|
+
+::
+
+    TRAIL_MODEL_ENABLED = True
+
+Show Trails on menu or not.
+
+|
+
+::
+
+    LANDEDGE_MODEL_ENABLED = True
+
+Show land on menu or not.
+
+|
+
+::
+
+   LAND_BBOX_CITIES_ENABLED = True
+   LAND_BBOX_DISTRICTS_ENABLED = True
+   LAND_BBOX_AREAS_ENABLED = False
+
+Show filter bbox by zoning.
+
+.. image:: images/options/zoning_bboxs.png
+
+**Translations**
+::
+
+    LANGUAGE_CODE = 'fr'
+
+Language of your interface.
+
+**Geographical CRUD**
+::
+
+    PATH_SNAPPING_DISTANCE = 2.0
+
+Minimum distance to merge 2 paths in unit of SRID
```

Comment:
to merge 2 paths automatically ?

Reply:
Que veux tu dire ?

Reply:
Voir mon commentaire ci-dessous, j'ai pas bien compris ce paramètre